script/Ciel/Day 6/06_01_CIEL06_7.txt

```diff
@@ -223,14 +223,14 @@ His expression is more than simply anguished. It's closer to hatred.
 -- そっか。そうね、今までのわたしなら、確かにもうちょっと時間をかけたかも。でも今回は一秒も無駄にしたくなかったの。ロアなんかに使う時間がもったいなくて」
 I see.
  Yeah, normally I probably would have taken a little longer.
- But this time around I didn't want to waste even a single second.
+ But this time around, I didn't want to waste even a single second.
  Roa isn't worth my time."
 }
 [sha:50658100157dab4b2b1b8764be203ed601ee980c]{
 -- Page 268, Offset 13864.
 -- 「ただの気まぐれかよ。なんだそれ。何か悪いモンでも食べたのか？　それとも故障が近いのか？　精密機械が気分でパターン変えてんじゃねえよ、クソ！」
 "You did this on a whim?!
- Fuck off.
+ Fuck off!
```

Comment:
is this change based on the va? from the text it looks like a curt "fuck off."

Unless the va is quite angry, I'd revert

Reply:
Yeah this is a 'don't be ridiculous' not an imperative

Reply:
Yeah it was. I double checked and I change my mind. The exclamation point doesn't fit that well